appengine/go11x/tasks/create_task/create_task.go

```diff
@@ -61,6 +61,9 @@ func createTask(projectID, locationID, queueID, message string) (*taskspb.Task,
 		return nil, fmt.Errorf("cloudtasks.CreateTask: %v", err)
 	}
 
+	// Closing the client is important, if not done then it will result in Memory spike
```

Comment:
I don't think we need the comment? I could be convinced, though.

Reply:
Sure yeah